core/src/test/java/org/mule/tck/testmodels/fruit/Stone.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.tck.testmodels.fruit;
+
+import org.mule.api.lifecycle.Initialisable;
+import org.mule.api.lifecycle.InitialisationException;
+
+import java.io.Serializable;
+
+public class Stone implements Serializable, Initialisable
+{
+    /**
+     * Serial version
+     */
+    private static final long serialVersionUID = -7631993371500076921L;
+
+    private Fruit fruit;
+
+    public Fruit getFruit()
+    {
+        return fruit;
+    }
+
+    public void setFruit(Fruit fruit)
+    {
+        this.fruit = fruit;
+    }
+
+    @Override
+    public void initialise() throws InitialisationException
+    {
+        // Nothing to do
```

Comment:
maybe makes sense to increase a counter or something that allows you to verify that this method was actually called?